docs/pbc/all/merchant-management/202410.0/marketplace/manage-in-the-back-office/manage-merchants-and-merchant-users-in-the-back-office.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
This page was carried over from previous versions, with a new link added at the bottom.